proposals/lip-0048.md

```diff
@@ -78,43 +68,37 @@ The Fee module does not expose any functions.
 
 #### Verify Transaction
 
-During `trs` verification, the following logic is applied:
-
 ```python
-minFee = MIN_FEE_PER_BYTE * size(trs)
-         + extraCommandFee(trs.moduleID, trs.commandID)
-if trs.fee < minFee:
-     trs is invalid
+def verify(trs: Transaction) -> None:
+    minFee = MIN_FEE_PER_BYTE * size(trs)
+    if trs.fee < minFee:
+         raise Exception("Insufficient transaction fee")
 ```
 
 #### Before Transaction Execution
 
-Before a transaction `trs` is executed, the following logic is applied:
-
 ```python
-minFee = MIN_FEE_PER_BYTE * size(trs)
-         + extraCommandFee(trs.moduleID, trs.commandID)
-senderAddress is derived from trs.senderPublicKey
-generatorAddress is the address of the generator of the block including trs
-
-if TOKEN_ID_FEE.chainID == 0: # Fee token is a native token
-    token.burn(senderAddress,
-               TOKEN_ID_FEE,
-               minFee)
-    token.transfer(senderAddress,
-                   generatorAddress,
-                   TOKEN_ID_FEE,
-                   trs.fee - minFee)
-else:
-    token.transfer(senderAddress,
-                   generatorAddress,
-                   TOKEN_ID_FEE,
-                   trs.fee)
+def beforeTransactionsExecute(trs: Transaction, b: Block) -> None:
+    minFee = MIN_FEE_PER_BYTE * size(trs)
+    senderAddress = SHA256(trs.senderPublicKey)[:20]
```

Comment:
In other LIPs we used: 

`senderAddress = Address derived from trs.senderPublicKey`

Your formulation is also fine for me, but maybe we could align on this (I would use `senderAddress = SHA256(trs.senderPublicKey)[:ADDRESS_LENGTH]`)

Reply:
fine for me